tensorflow/core/api_def/base_api/api_def_MatrixDeterminant.pbtxt

```diff
@@ -17,5 +17,12 @@ END
 The input is a tensor of shape `[..., M, M]` whose inner-most 2 dimensions
 form square matrices. The output is a tensor containing the determinants
 for all input submatrices `[..., :, :]`.
+
```

Comment:
Hi I think this is part of another pull request as well, correct me if I'm wrong😊